lib/webdriveragent.js

```diff
@@ -70,6 +70,7 @@ class WebDriverAgent {
     this.wdaLaunchTimeout = args.wdaLaunchTimeout || WDA_LAUNCH_TIMEOUT;
     this.usePreinstalledWDA = args.usePreinstalledWDA;
     this.xctestApiClient = null;
+    this.doNotAddXctrunnerSuffix = args.doNotAddXctrunnerSuffix;
```

Comment:
how about changing this cap to `updatedWDABundleIdSuffix` and have it equal to `.xctrunner` by default?
this way it would clearly be visible to where this capability belongs

Reply:
ah, it sounds good idea